### PR TITLE
Sweets/init

### PR DESCRIPTION
**Pull request by SweetmanTech**
# Mint Page for Founders Pass
- Burn 88 Relics (Polygon)
- Mint 1 Founders Pass (Ethereum)

Screenshot 2023-05-04 at 6 47 11 PM

### CODE REVIEW

File: components/Header/Header.tsx [hunk 1]
```diff
@@ -59,7 +57,10 @@ const Header: FC<HeaderProps> = ({ connect, logout }) => {
             )}
           </div>
         </Link>
-        <Link href="https://cre8ors.beehiiv.com/?_gl=1*16y8c82*_ga*NzQ0MjY2NzkyLjE2NzI4NzY1OTY.*_ga_E6Y4WLQ2EC*MTY4MTQ5NTA5Ny4zNS4xLjE2ODE0OTUxMDAuNTcuMC4w">
+        <Link
+          target="_blank"
```
Comment (SweetmanTech) [inline components/Header/Header.tsx:61]: this isn't doing anything and should be deleted
```suggestion
```

File: components/PendingTransactionModal/MintButton.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        minter: address,
+        maxMint: 1,
+        mintPrice: 0,
+      }
+      const merkleProof = generateMerkleProof(bytes32Array, userEntry)
+      const maxQuantity = isTestnet ? 100 : 1
+      const tx = await contract.purchasePresale(1, maxQuantity, 0, merkleProof)
+      await tx.wait()
+      setClaimStatus("completed")
+      onMint?.()
```
Comment (crazzywizard) [inline components/PendingTransactionModal/MintButton.tsx:38]: why is this conditional?

File: components/ClaimButton/ClaimButton.tsx [hunk 3]
```diff
@@ -86,12 +63,12 @@ const ClaimButton: FC<ClaimButtonProps> = ({ contractAddress, abi }) => {
...
                     />
                   ) : (
-                    "Claim Audio Trait"
+                    "Collect Founders Pass"
                   )}
                 </button>
               )
             })()}
-            {startConfetti && <Confetti width={width} height={height} />}
+            <PendingTransactionModal isOpen={isOpen} closeModal={() => setIsOpen(false)} />
```
Comment (crazzywizard) [inline components/ClaimButton/ClaimButton.tsx:71]: why not make this conditional and remove the `isOpen return null` in  PendingTransactionModal component?
`{isOpen ? <PendingTransactionalModal closeModal={()=>setIsOpen(false)}> :nulll}`

**Review by crazzywizard — APPROVED** [review]
LGTM! minor comments

File: components/ClaimButton/ClaimButton.tsx (hunk 3 above)
Comment (SweetmanTech) [reply components/ClaimButton/ClaimButton.tsx:71]: you got it 👍 

any reason why you prefer that?

would love to learn

Comment (SweetmanTech) [reply components/ClaimButton/ClaimButton.tsx]: Screenshot 2023-05-05 at 7 18 01 PM

**Pull request merged**